inventories/ec2/group_vars/all.yaml

```diff
@@ -13,12 +14,13 @@ regions:
     scylla_ami:  ami-72196418 # scylla 0.11
     loadgen_image: ami-d05e75b8 # ubuntu 14.04
     ubuntu_image: ami-d05e75b8 # ubuntu 14.04
+    fedora_ami: ami-a51564c0 # fedora 22
     key_name: tzach-virginia
     security_group: cassandra-security-group
 
 # list of only the first region
 first_regions:
-  us-west-2: "{{regions['us-west-2']}}"
+  us-east-1: "{{regions['us-east-1']}}"
```

Comment:
I have tested this patch - manually editing to fix the region issue, and it
works.

Thanks Tzach!

On Mon, Nov 9, 2015 at 1:44 PM, Tzach Livyatan <notifications@github.com>
wrote:

> In inventories/ec2/group_vars/all.yaml
> <https://github.com/scylladb/cassandra-test-and-deploy/pull/28#discussion_r44270490>
> :
>
> >      key_name: tzach-virginia
> >      security_group: cassandra-security-group
> >
> >  # list of only the first region
> >  first_regions:
> > -  us-west-2: "{{regions['us-west-2']}}"
> > +  us-east-1: "{{regions['us-east-1']}}"
>
> Glitch. I will fix.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scylladb/cassandra-test-and-deploy/pull/28/files#r44270490>
> .
>